### PR TITLE
Fix: Granger Fails To Call In Carpet Bomber In Generals' Challenge

### PR DESCRIPTION
**Pull request by commy2**
At the first wave he's attacking the players base, he is supposed to call in a Carpet Bomber, but he lacks the genpower unlock for it. This worked in Zero Hour 1.00, because no unlock was required to call in the Carpet Bomber.

This patch unlocks the Carpet Bomber for him.

US Military spending at its best.

### CODE REVIEW

**Review by xezon — DISMISSED** [review]

File: Patch104pZH/GameFilesEdited/Maps/GC_AirGeneral/map.ini [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Object AirfieldHanger01
+ RemoveModule ModuleTag_03
+END
+
+Object CivilianAirport
+ RemoveModule ModuleTag_05
+END
+
+Object BaikonurGroundControlCenter
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Maps/GC_AirGeneral/map.ini:25]: What are these 3 objects here for?

Comment (commy2) [reply Patch104pZH/GameFilesEdited/Maps/GC_AirGeneral/map.ini:25]: That's the map.ini of this map. Additions are at the end of the file.

I believe the BGCC is a prop in that mission.

**Comment by xezon** [comment]
Oh. May I suggest to first add the unedited map.ini and then add the edit on the next commit? This way the diff can be preserved in main branch.

**Comment by commy2** [comment]
We could add all the 1.04 map.ini files from Maps folder / MapZH.big.

**Comment by Jundiyy** [comment]
I'm happy to bring back the Carpet. I remember it vanishing after the update.

**Comment by xezon** [comment]
Merge #895 first.

**Pull request closed without merging**